src/material/datepicker/datepicker-content.scss

```diff
@@ -28,7 +33,10 @@ $touch-max-height: 788px;
 
   .mat-calendar {
     width: $non-touch-calendar-width;
-    height: $non-touch-calendar-height;
+    min-height: $non-touch-calendar-height;
+
+    // Allow calendar to grow to a slightly larger height to accomodate WCAG 1.4.12 text spacing.
+    max-height: $non-touch-calendar-wcag-text-spacing-height;
```

Comment:
As it's written right now, this also applies to the calendar in `touchUI` mode which might cause some issues. This is where it is being cut off: 
<img width="699" alt="image" src="https://user-images.githubusercontent.com/4450522/191468680-5cfeead5-b8e6-4a6f-a1b8-2fe44eeb7f77.png">

Reply:
shoot, I thought the `$non-touch-calendar-height` only affects desktop :). I can take another look at this later to see if there's a different way to do this.